SRC/icbacn.F90

```diff
@@ -7,7 +7,7 @@ subroutine cnaupd_c(ido, bmat, n, which, nev, tol, resid, ncv, v, ldv,&
   implicit none
 #include "arpackdef.h"
   integer(kind=c_int),                              intent(inout) :: ido
-  character(kind=c_char),       dimension(1),       intent(in)    :: bmat
+  character(kind=c_char),                           intent(in)    :: bmat
   integer(kind=c_int),          value,              intent(in)    :: n
```

Comment:
Kill `dimension(1)` : sounds good if CI is OK ! Could you to that also for parpack (PARPACK/SRC/MPI/icbp*F90) ?

Reply:
Ok! Done